src/model/paste/DraftPasteProcessor.js

```diff
@@ -46,6 +47,29 @@ const DraftPasteProcessor = {
     html: string,
     blockRenderMap?: DraftBlockRenderMap,
   ): ?{contentBlocks: ?Array<BlockNodeRecord>, entityMap: EntityMap} {
+    const body = getSafeBodyFromHTML(html);
+    const fragmentElt = body && body.querySelector('[data-editor-content]');
+    const fragmentAttr =
+      (fragmentElt && fragmentElt.getAttribute('data-editor-content')) || null;
+
+    // Handle the paste without converting the HTML if it comes from another Draft.js editor.
+    if (fragmentAttr) {
+      let rawContent;
+
+      try {
+        // If JSON parsing fails, handle the paste as normal HTML.
+        rawContent = JSON.parse(fragmentAttr);
```

Comment:
I'm not sure in what scenario that would occur – perhaps if the clipboard gets truncated somehow.